### DISCUSSION TITLE
Formuler Z11 Pro Max

### DISCUSSION THREAD

**Discussion by coasttech** [PRProd/HA-Firemote#261] [New Device Family Support Request]
I own this and have almost everything working. I'm wondering what I could do to bring official device support on board?

https://www.formuler.tv/z11promax

**Comment by PRProd** [comment]
Greetings @coasttech!

Right now, the process for requesting support for new Device Families is to visit discussion #41 and to cast your vote.  Since that specific family of devices is not yet on the poll, you'll want to vote for "other" and write the details in the discussion thread.

The last few device family additions have been based off of the number of votes received in the poll at discussion #41, but future additions to Firemote might be also influenced by my ability to purchase the devices in my region (US) and/or their cost.  I personally have been buying the devices so I can properly test them 100% myself before publishing support.  It looks like this device is available on Amazon, so I've added it to my [Amazon Firemote Wish List](https://www.amazon.com/hz/wishlist/ls/2UDS10RPJ6F5K?ref_=wl_share) in case someone chooses to send one to me as a gift.

For now, I will close this discussion.  Once support for this device starts being developed, I will open a beta testing discussion group to receive your testing input. When it's ready, we will release it to the public!

**Reply by coasttech** [reply]
kk

**Comment by PRProd** [comment]
My sincere apologies.  I forgot to add this to [the official poll](https://github.com/PRProd/HA-Firemote/discussions/41) after the latest poll reset. 

You can still vote for the [Formuler device family](https://www.formuler.tv/) by voting for **Other** and writing **Formuler** in the [proper thread](https://github.com/PRProd/HA-Firemote/discussions/41#discussioncomment-5293057),
